plugin/file/secondary.go

```diff
@@ -138,15 +147,21 @@ Restart:
 
 			ok, err := z.shouldTransfer()
 			if err != nil {
+				UpdatedSerial.WithLabelValues(server, z.origin).Dec()
```

Comment:
Isn't this supposed to be gauge? It seems to be used as a counter and in lockstep with `UpdatedSuccess`. Is this intended?

Reply:
I got confused initially about what this metric meant. I will be correcting this in my next push